src/types/TodoStatus.ts

```diff
@@ -0,0 +1,5 @@
+export enum TodoStatus {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
+}
```

Comment:
Todo names usually start from capital letter
```suggestion
export enum TodoStatus {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
}
```